src/caret_analyze/plot/visualize_lib/bokeh/bokeh.py

```diff
@@ -44,6 +47,239 @@ class Bokeh(VisualizeLibInterface):
     def __init__(self) -> None:
         pass
 
+    def stacked_bar(
+        self,
+        metrics,
+        xaxis_type: str,
+        ywheel_zoom: bool,
+        full_legends: bool,  # NOTE: unused because LegendManager not apply stacked bar.
+    ) -> Figure:
+
+        # NOTE: relation between stacked bar graph and data struct
+        # # data = {
+        # #     a : [a1, a2, a3],
+        # #     b : [b1, b2, b3],
+        # #     'start time': [s1, s2, s3]
+        # # }
+        # # y_labels = [a, b]
+
+        # # ^               ^
+        # # |               |       ^       [] a
+        # # |       ^       |       |       [] b
+        # # |       |       a2      |
+        # # |       a1      ^       a3
+        # # |       ^       |       ^
+        # # |       |       |       |
+        # # |       b1      b2      b3
+        # # +-------s1------s2------s3---------->
+
+        # # get stacked bar data
+        data: Dict[str, list[int]] = {}
+        y_labels: List[str] = []
+        title: str = 'Stacked bar graph'
+        y_axis_label = 'latency [ms]'
+        target_objects = metrics.target_objects
+        data, y_labels = metrics.to_stacked_bar_dict()
+
+        fig = self._init_figure(title, ywheel_zoom, xaxis_type, y_axis_label)
+        records_range = Range([target_objects.to_records()])
+        frame_min, frame_max = records_range.get_range()
+        x_label = 'start time'
+        if xaxis_type == 'system_time':
+            self._apply_x_axis_offset(fig, frame_min, frame_max)
+        elif xaxis_type == 'index':
+            x_label = 'index'
+        else:  # sim_time
+            raise NotImplementedError()
+
+        color_selector = ColorSelectorFactory.create_instance(coloring_rule='unique')
+        legend_manager = LegendManager()
+        stacked_bar_source = StackedBarSource(legend_manager, target_objects, 0.0, xaxis_type)
+        fig.add_tools(stacked_bar_source.create_hover())
+        stacked_bar_data, x_width_list = \
+            self._get_stacked_bar_data(data, y_labels, xaxis_type, x_label)
+        bottom_labels = self._get_bottom_labels(y_labels)
+        bottom_labels = bottom_labels[1:]
+        source = stacked_bar_source.generate(
+            target_objects,
+            stacked_bar_data,
+            y_labels,
+            bottom_labels,
+            x_width_list
+        )
+
+        for y_label, bottom in zip(y_labels[:-1], bottom_labels):
+            color = color_selector.get_color(y_label)
+            renderer = fig.vbar(
+                x=x_label,
+                top=y_label,
+                width='x_width_list',
+                source=source,
+                color=color,
+                bottom=bottom,
+                # legend_label=y_label,
+            )
```

Comment:
Is this a necessary comment?
Line 123, 131, too.
```suggestion
                color=color,
                bottom=bottom
            )
```